socrata/socrata_upload.py

```diff
@@ -10,6 +10,14 @@
 from pyathena import connect
 from pyathena.pandas.cursor import PandasCursor
 
+# Create a session object so HTTP requests can be pooled
+s = requests.Session()
+s.verify = True
```

Comment:
**[Nitpick, non-blocking]** I think this should be [`True` by default](https://docs.python-requests.org/en/latest/api/#requests.Session.verify), no?